src/content/popup/KanjiReferencesTable.tsx

```diff
@@ -0,0 +1,115 @@
+import { KanjiResult } from '@birchill/jpdict-idb';
+
+import { ReferenceAbbreviation, getSelectedReferenceLabels } from '../../common/refs';
+import { getReferenceValue } from '../reference-value';
+import { I18nProvider, useLocale } from '../../common/i18n';
+import { useMemo } from 'preact/hooks';
```

Comment:
Nit: I tend to put the 3rd-party imports in a separate group at the top (but clearly I haven't added an eslint/prettier rule for that yet 😅).